.github/actions/lint-test-chart/action.yml

```diff
@@ -0,0 +1,61 @@
+name: Lint test chart
+description: "Run lint test chart"
+
+runs:
+  using: "composite"
+  steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.9"
+          check-latest: true
+
+      - name: Chart Rendering Tests
+        shell: bash
+        run: |
+          go test -v ./contrib/charts/dragonfly/...
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.6.1
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        shell: bash
+        run: |
+          changed=$(ct list-changed --config .ct.yaml)
+          if [[ -n "$changed" ]]; then
+            echo "changed=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Run chart-testing (lint)
+        shell: bash
+        run: |
+          ct \
+            lint \
+            --config .ct.yaml \
+            ${{github.event_name == 'workflow_dispatch' && '--all'}} ;
```

Comment:
I kept this because if the jobs gets triggered manually it will become true. I am not sure about the ` &&  `-all` part, I am running the reg tests manually to see how it affects it and I will remove it if it's not used.

Reply:
yep we need it. see https://github.com/helm/chart-testing/blob/main/doc/ct_lint.md and `--all`